src/components/common/PlotlyCard/styles.js

```diff
@@ -1,9 +1,29 @@
-export const mobileStyles = {};
+import { isMobile, isTablet, isDesktop } from '../../../utils/helpers';
+// this is a styles function
+// requires 'display' arg (string)
+// optional  'theme' arg (if you need to access redux state)
```

Comment:
Great comments!

Reply:
I felt like it was important because of how unorthodox passing styles like this felt. 
Styling in here is very limited due to the fact we had no access to pseudo selectors. 
We had to use less files to adjust those styles but I didn't know we were allowed to use the styled components library in conjunction with antdesign.

Implementing styled components would make it possible to get rid of almost all less files and creating a more consistent design pattern throughout the app.